arch/arm/src/a1x/hardware/a10_memorymap.h

```diff
@@ -502,7 +502,7 @@
  *                                    LI PAGE TABLE
  *   ADDRESS RANGE           SIZE     ENTRIES       SECTIONS
  *   ----------------------- ------- -------------- ---------
- *   0xc000:0000-0xffef:ffff 1022MB  *0x3000-0x3ff8 1022
+ *   0xc000:0000-0xffef:ffff 1022MB   0x3000-0x3ff8 1022
```

Comment:
Please fix inclusion of board_memorymap.h

Reply:
Can't from the comment:
```
/* A1X Virtual (mapped) Memory Map
 *
 * board_memorymap.h contains special mappings that are needed when a ROM
 * memory map is used.  It is included in this odd location because it
 * depends on some the virtual address definitions provided above.
 */

#include <arch/board/board_memorymap.h>
```